Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -10,15 +10,21 @@ public struct SwiftPackageManagerDependencies: Codable, Equatable {
 
     /// Set of deployment targets to be used when the SPM package does not specify a target version.
     public let deploymentTargets: Set<DeploymentTarget>
+    
+    public let productSettings: [String: SettingsDictionary]
 
     /// Creates `SwiftPackageManagerDependencies` instance.
     /// - Parameter packages: List of packages that will be installed using Swift Package Manager.
     /// - Parameter productTypes: The custom `Product` types to be used for SPM targets.
     /// - Parameter deploymentTargets: Set of deployment targets to be used when the SPM package does not specify a target version.
-    public init(_ packages: [Package], productTypes: [String: Product] = [:], deploymentTargets: Set<DeploymentTarget> = []) {
+    public init(_ packages: [Package],
```

Comment:
It's important that we document all the changes in this file because it's a user-facing model and Xcode might end up showing outdated documentation to the users.